the-rules/data/no-rules.json5

```diff
@@ -7,5 +7,13 @@
   // You may want to know why someone needs branches if there are no rules, well its a story about time
   // if I had more time I would explain it, but I'm still working on the rules of math, which are not so much rules as ...
   // sorry that may have been a rant there, keep going, and just define your own truth.
+  ruleOne: {
+    active: true,
+    parameters : [
+     "many different string statements allowed",
+     "we can propose any type of rule, like a rule to allow rules into the repo based on consensus",
+     "typically this should contain single statements of fact, with one instance of it in action as an example",
```

Comment:
I like the idea of having a single fact, and I thought about adding the rules as facts and using a language parser for the inputs, and maybe playing with LLVM's. I also thought about using word orgins and having the game have modes. In the end I went with just getting something out fast.

I think one of the hard parts about getting a game out is engagement, and doing quick small and fun stuff. That is for sure a core goal with this.